REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.6
-NLPModels 0.4.1
+NLPModels
```

Comment:
It's recommended to specify a version here. Maybe we need a new release?

Reply:
Yes, I deleted the version to see if the tests passed. But the continuous integration install the last tagged version of ```NLPModels``` even so.